### PR TITLE
Update the specification of Foundry address scheme

### PR DESCRIPTION
**Pull request by sgkim126**

### CODE REVIEW

File: spec/JSON-RPC.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ A two-letter string to denote a network. For example, "cc" is for the main netwo
 
 ## PlatformAddress
 
-A string that starts with "(NetworkID)c", and Bech32 string follows. For example, "cccqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqz6sxn0" is for the main network, and "wccqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqz6sxn0" is for the Corgi test network. See [the specification](CodeChain-Address.md#1-platform-account-address-format).
+A string that starts with "(NetworkID)c", and Bech32 string follows. For example, "cccqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqz6sxn0" is for the main network, and "wccqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqz6sxn0" is for the Corgi test network. See [the specification](Foundry-Address.md).
```
Comment (majecty) [inline spec/JSON-RPC.md:28]: The description of PlatformAddress is deprecated. Could you update or remove it?

File: spec/Staking.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ makeKey(...fragments) = blake256(rlp([
 
  * Value: `rlp(list of account)`
 
-    An `account` is an [Account Id](./CodeChain-Address.md) which is a blake160 hash of a public key.
+    An `account` is an [address](./Foundry-Address.md) which is a blake160 hash of a public key.
```
Comment (majecty) [inline spec/Staking.md:24]: ditto.

File: spec/JSON-RPC.md (hunk 1 above)
Comment (sgkim126) [reply spec/JSON-RPC.md:28]: fixed

File: spec/Staking.md (hunk 2 above)
Comment (sgkim126) [reply spec/Staking.md]: fixed

**Pull request merged**